test/types/scalar/hilde/absMinInt.bad

```diff
@@ -0,0 +1 @@
+absMinInt.chpl:12: error: halt reached - abs should never return a negative number.
```

Comment:
Should this check and halt be moved into abs() itself?  (potentially only when some form of checks is on?)

Reply:
That's an interesting question, but I think it should be separate from this PR. (All I did was add a .bad file for the normal failure mode for this test; other changes to the test just had to do with keeping the types what they were after the changes to things like `1:int(8) + 2`.). Feel free to open an issue about that - or I can if you consider it important to do that for this PR.

Reply:
:+1: